pkg/docker/fast_push.go

```diff
@@ -128,12 +146,30 @@ func startUploadURL(objectType string, digest string) url.URL {
 	return uploadUrl
 }
 
-func uploadFile(objectType string, digest string, path string, token string) error {
+func verificationURL(objectType string, digest string, uuid string) url.URL {
+	verificationUrl := baseURL()
+	verificationUrl.Path = strings.Join([]string{"", "uploads", objectType, "sha256", digest, uuid, "verification"}, "/")
+	return verificationUrl
+}
+
+func checkVerificationStatus(req *http.Request, client *http.Client) (bool, error) {
+	checkResp, err := client.Do(req)
+	if err != nil {
+		return false, err
+	}
+	defer checkResp.Body.Close()
+	return checkResp.StatusCode == http.StatusOK, nil
```

Comment:
I think this will need to look at the body, too, since completed != verified.

Here's where the e2e test script does it, fwiw: https://github.com/replicate/monobeam/blob/1fb65095db29e1b07368e0e09cd544735b033805/script/e2e-test#L92-L108

Reply:
Resolved in [e7ff752](https://github.com/replicate/cog/pull/2128/commits/e7ff752774997f3282db15b69eee7f5267f96ac8)